src/lib/config.ts

```diff
@@ -68,28 +81,32 @@ const loadConfig = async () => {
 };
 
 export const persistPeers = async () => {
-  await ensurePeersDirectoryExists();
+  await ensureDirectoryExists(peersFilePath);
   await fs.writeFile(peersFilePath, JSON.stringify(config.peers, null, 2));
 };
 
+export const persistDestinations = async () => {
+  await ensureDirectoryExists(destinationsFilePath);
+  await fs.writeFile(destinationsFilePath, JSON.stringify(config.destinations, null, 2));
+};
 
-const ensurePeersDirectoryExists = async () => {
+const ensureDirectoryExists = async (directory: string) => {
   try {
-    await fs.access(peersFilePath);
+    await fs.access(directory);
   } catch(err: any) {
     if(err.code === 'ENOENT') {
-      await createPeersDirectory();
+      await createPeersDirectory(directory);
     } else {
-      log.warn(`Could not check for existence of peers subdirectory ${err.code}`);
+      log.warn(`Could not check for existence of ${err.code}`);
     }
   }
 };
 
-const createPeersDirectory = async () => {
+const createPeersDirectory = async (directory: string) => {
```

Comment:
should we rename this to just `createDirectory`, since we can now pass in any arbitrary directory

Reply:
Great point, the other functions were renamed to make them generic, this one was missed.